README.md

```diff
@@ -1,7 +1,17 @@
 <img src="figs/logo_01.png" width="150"/>
 
+# ANNOUNCEMENT:
+**Snorkel v0.9 is being released this summer** and we are absolutely certain you're going to love the new version!
 
-**_v0.7.0-beta_**
+Why is it 0.9 when the last version was 0.7? Because this is more than just an incremental change. It's a total rewrite from the ground up with portability, scalability, and extensibility in mind, including full unit test coverage, type checking, upgraded docstrings, and a fresh batch of tutorials demonstrating different use cases and integrations! The new version will be available via `pip` and `conda` but **will not be backwards compatible**. The current (v0.7) Snorkel code will be moved to another repository to continue to support existing applications that depend on it.
```

Comment:
Two comments here:

- Maybe "full refactor" rather than "total rewrite", to suggest that we're not changing the core concepts
- I would list the clear bullet points about conceptual differences _before_ "portability...", i.e.:
    * Support for new operators (LFs, TFs, SFs)
    * New matrix completion-style method for learning the LF accuracies and correlation structure
    * More modular form factor, with application-specific data management (for e.g. information extraction) moved to separate repos
    * Native MTL support
    * And much more!

Not verbatim, but something like the above first before the other changes

Reply:
* "Native MTL support" might be difficult to interpret (generative vs. discriminative)
* new operators __for training data__
* do we want to say something about 101 guides here?